graphqlapi/src/main/java/io/stargate/graphql/persistence/schemafirst/SchemaSourceDao.java

```diff
@@ -39,21 +40,23 @@
 // TODO purge old entries
 public class SchemaSourceDao {
 
-  public static final String TABLE_NAME = "graphql_schema";
-  @VisibleForTesting static final String KEY_COLUMN_NAME = "key";
+  public static final String KEYSPACE_NAME = "stargate_graphql";
```

Comment:
In the auth module the keyspace name is configurable. I don't see a strong need for it here, but if necessary we can always add it later.